lib/Fhp/FinTsNew.php

```diff
@@ -138,6 +128,21 @@ public function persist(bool $minimal = false)
         ]);
     }
 
+    public function loadPersistedInstance($persistedInstance)
```

Comment:
Diese Funktion braucht phpDoc und eine Typ-Angabe für den Parameter. Und der Satz "need to be passed to the constructor" bei `persist()` muss vmtl. auch noch erweitert werden.

Reply:
Danke. Hab ich gemacht.